community/values/src/test/java/org/neo4j/values/storable/BufferValueWriter.java

```diff
@@ -157,7 +156,7 @@ public void writeString( char value )
     @Override
     public void writeUTF8( byte[] bytes, int offset, int length ) throws RuntimeException
     {
-        buffer.add( UTF8.decode( bytes, offset, length ) );
+        buffer.add( new String( bytes, offset, length, StandardCharsets.UTF_8 ) );
```

Comment:
Interesting... what is the reason?

Reply:
values doesn't doesn't have that utility as a dependency anymore. `UTF8.decode` just does that very thing.